pkg/cmd/ctlstore/main.go

```diff
@@ -86,6 +86,7 @@ type supervisorCliConfig struct {
 	ReflectorConfig     reflectorCliConfig `conf:"reflector" help:"reflector configuration"`
 	Shadow              bool               `conf:"shadow" help:"set this to true to emit shadow=true metric tags"`
 	Dogstatsd           dogstatsdConfig    `conf:"dogstatsd" help:"dogstatsd Configuration"`
+	MaxLedgerLatency    time.Duration      `conf:"minimum-ledger-latency" help:"Minimum ledger latency, at which the supervisor halts snapshots and waits for the ledger to catch up."`
```

Comment:
The `conf` tag here should be `maximum-ledger-latency` (or `min-ledger-latency` if you're into brevity). also the `help` tag should be adjusted similarly.